src/test/java/com/faforever/client/chat/AbstractChatTabControllerTest.java

```diff
@@ -331,4 +333,18 @@ public void testChannelNamesTransformedToHyperlinks() {
     String output = instance.replaceChannelNamesWithHyperlinks("Go to #moderation and report a user");
     assertThat(output, is("Go to <a href=\"javascript:void(0);\" onClick=\"java.openChannel('#moderation')\">#moderation</a> and report a user"));
   }
+
+  @Test
+  public void testMentionPattern() {
```

Comment:
Some more suggestions to test. I’m not even sure what the desired behavior would be for these tho cuz they’re kinda weird:
- `-Box--`
- `-Box-Box-`

Maybe also:
- `[-Box-]`

Since that would have failed with my lazy A-z character class.